sdk/python/lib/pulumi/invoke.py

```diff
@@ -127,3 +134,70 @@ def merge(
         dest.version = dest.version if source.version is None else source.version
 
         return dest
+
+
+class InvokeTransformArgs:
+    """
+    InvokeTransformArgs is the argument bag passed to a resource transform.
+    """
+
+    token: str
+    """
+    The token of the Invoke.
```

Comment:
Feel like we should be consistent with the capitalisation of `Invoke`/`invoke` -- don't mind which.